src/features/bulk/FileViewer/IndividualFieldsForm.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { Heading } from '@amsterdam/asc-ui';
+
+import { useAppDispatch, useAppSelector } from '../../hooks';
+import { createSchemaFromMetadataProps, createUISchemaCompactFromMetadataProps } from '../../../utils';
+import { MetadataGenericType } from '../../../types';
+import Form from '../../../components/Form/Form';
+
+import { IBulkField, IBulkFile, IBulkFileMetadata } from '../bulk/store/model';
+import {
+	convertBulkFieldsToBulkFileMetadata,
+	convertBulkFieldsToMetadataProperties,
+	convertBulkFileMetadataToMetadataGenericTypes,
+	identicalObjects,
+	reduceMetadata,
+} from '../bulk/utils';
+import { IndividualFieldsFormStyle } from './styles';
+import { setFileMetadata } from '../bulk/store/slice';
+import { getFieldsForFile } from '../bulk/store/selectors';
+
+export type Props = {
+	fieldData: any;
+	fields: IBulkField[];
+	file: IBulkFile;
+	onChange: (data: MetadataGenericType, valid: boolean) => void;
+};
+
+const getMergedData = (fields: IBulkField[], fileFields: IBulkFileMetadata[] | undefined) =>
+	convertBulkFileMetadataToMetadataGenericTypes(
+		reduceMetadata(convertBulkFieldsToBulkFileMetadata(fields), fileFields),
+	);
+
+export default function IndividualFieldsForm({ ...props }: Props) {
```

Comment:
Can be safely destructured on the previous line. The rest operator has no use here.

Reply:
check